Update rake, remove unnecessary parameter

DerekMelchin opened the pull request.

DerekMelchin commented:
That first commit is supposed to be for episode 13. I'm not sure how it got added here.

wndxlori commented:
I think there's still a problem in the "before" code.
https://github.com/wndxlori/MotionInMotionSourceCode/blob/0f98ed4eea5d5d3d8b5ec048650dedc13252c5a6/Episode-13/before/Blapp/app/controllers/PseudonymController.rb#L32

Also, I just noticed your branch name is ep 14, but the code is on Ep 13 dir?

DerekMelchin commented:
This branch was just a mess. I fixed it with following pull requests. As for episode 14 though, I found a bug that I can't get around, so I updated it's status in the spread sheet.
